tests/pipelines/main.nf.test

```diff
@@ -49,6 +54,61 @@ nextflow_pipeline {
         }
     }
 
+    test("Small-scale test of full pipeline with multiple queries"){
+        tag "pipeline_success_multiple_queries"
+
+        when{
+            params {
+                input = "$baseDir/tests/data/samplesheets/samplesheet-multiple_queries.csv"
+                outdir = "results"
+            }
+        }
+
+        then {
+            assert workflow.success
+            assert path("$launchDir/results").exists()
+
+            // Check merged profiles
+            def actual_profile_ref = path("$launchDir/results/locidex/merge/reference/merged_ref/merged_profiles_ref.tsv")
+            def expected_profile_tsv = path("$baseDir/tests/data/profiles/expected-profile_queries1.tsv")
+            assert actual_profile_ref.text == expected_profile_tsv.text
+
+            // Check query profiles
+            def actual_profile_query = path("$launchDir/results/locidex/merge/query/merged_value/merged_profiles_value.tsv")
+            def expected_profile_query_tsv = path("$baseDir/tests/data/profiles/expected-profile_queries2.tsv")
+            assert actual_profile_query.text == expected_profile_query_tsv.text
+
+            // Check computed pairwise distances
+            def actual_distances = path("$launchDir/results/distances/results.text")
+            def expected_distances = path("$baseDir/tests/data/distances/expected_pairwise_queries_dists.txt")
+            assert actual_distances.text == expected_distances.text
+
+            // Verify cluster file
+            def actual_cluster = path("$launchDir/results/cluster/expected_clusters.txt")
```

Comment:
Is it the case that the output filename for the clusters is `expected_clusters.txt`?

Reply:
Yes it is :)

Reply:
Although ... it is a little confusing to call the output file for the reference clusters this way.
I have changed the output filename to: `reference_clusters` in the `cluster_file` process and updated the tests.

See changes here: [15b7090](https://github.com/phac-nml/gasnomenclature/pull/14/commits/15b7090691f90dbb969be9db01218715673201ca)